vcluster/_fragments/sleepmode-istio-example.mdx

```diff
@@ -0,0 +1,116 @@
+import Flow, { Step } from '@site/src/components/Flow';
+import CodeBlock from '@theme/CodeBlock'
+import SleepmodeIstioConfig from '!!raw-loader!@site/vcluster/configure/vcluster-yaml/_code/sleepmode-istio-config.yaml'
+import SleepmodeIstioExample from '!!raw-loader!@site/vcluster/configure/vcluster-yaml/experimental/_code/sleepmode-istio-examples.yaml'
+
+### Configure sleep mode for use with Istio
+
+<Flow id="sleepmode-ingress-example">
+  <Step>
+Create the `kind` cluster.
+
+```shell title="Create a kind cluster"
+kind create cluster --name istio-demo --config - <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    extraPortMappings:
+      - containerPort: 30000
+        listenAddress: 127.0.0.1
+        hostPort: 80
+        protocol: TCP
+      - containerPort: 31000
+        listenAddress: 127.0.0.1
+        hostPort: 443
+        protocol: TCP
+      - containerPort: 32000
+        listenAddress: 127.0.0.1
+        hostPort: 15021
+        protocol: TCP
+EOF
+```
+  </Step>
+  <Step>
+    Install istio
+
+    ```shell title="Install istio"
+    istioctl install -y
+    ```
+
+    or
+
+    ```shell title="Install with ambient"
+    istioctl install --set profile=ambient --set "components.ingressGateways[0].enabled=true" --set "components.ingressGateways[0].name=istio-ingressgateway" --skip-confirmation
```

Comment:
How come only ambient has ingressGateway turned on ? Wouldn't it be on for both ambient and not

Reply:
A result of trying different things to get things working with kind and landing on the first one that worked.

Reply:
haha, ok, but then are you saying that the first one (sidecar) worked without these flags ?

Reply:
Correct.  If I just do the standard install sleeping and waking work fine.

Reply:
If it doesn't matter, wouldn't it also make sense to add it to one above too ? That way it would work all the way from outside of cluster for both options.